go.mod

```diff
@@ -11,4 +11,5 @@ require (
 require (
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	golang.org/x/sys v0.9.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```

Comment:
Make sure to run `go mod tidy`.

This isn't an indirect dependency since you're using it explicitly in `bake.go` and the `go` tooling should bump it up to the top level require block automatically.